qemu-test-rauc-config

```diff
@@ -13,6 +13,9 @@ events=all
 [keyring]
 path=ca.cert.pem
 
+[streaming]
```

Comment:
A note in the commit message why this is required as preparation would be helpful.